src/lib.rs

```diff
@@ -2,9 +2,9 @@
     cfg_attr(all(), doc = include_str!("../README.md")),
 )]
 
-extern crate proc_macro;
+extern crate macro_rules_attribute_proc_macro;
```

Comment:
This line oughtn't be needed anymore; it was already the workaround then to avoid the cross-compilation bug, but I guess that `resolver = "2"` breaks it 😄:

```suggestion
```